draft-ietf-webpush-protocol.xml

```diff
@@ -1084,7 +1084,7 @@ HEADERS      [stream 82] +END_STREAM
             A push service MAY expire a subscription set at any time which MUST also
             expire all push message subscriptions in the set. If a user agent has an
             outstanding request to a push subscription set (<xref target="monitor-set"/>)
-            this can be signaled by returning a 400-series status code, such as 410 (Gone).
+            this can be signaled by returning a 410 (Gone) status code.

```

Comment:
> Again, these are different failures. 404 is "push services gives up on delivery before TTL" and 410 is for expirations. The subscription set language should match the subscription language for expiration.

Yes, they are different failures because they are different resources, but they both reduce to "the thing that you were asking about went away".

> It's unclear whether you're proposing to simply return 410 for both cases - but that's not reflected in this pull request.

Yes, both would be 410.  We already mandate 410 elsewhere, this makes the one on which we prevaricate more definite.